pkg/kubelet/rkt/rkt.go

```diff
@@ -1320,6 +1393,13 @@ func (r *Runtime) KillPod(pod *api.Pod, runningPod kubecontainer.Pod) error {
 		return err
 	}
 
+	if !kubecontainer.IsHostNetworkPod(pod) {
+		if err := r.cleanupPodNetwork(&runningPod, true); err != nil {
```

Comment:
@dcbw This takes care of the network cleanup when we explicitly kill/stop the pods. But what if the pod is a crash loop, and is deleted by garbage collection?
Actually, I don't find any place in dockertools that take care of the network when the pod is garbage collected today, did I miss something?